.rubocop.yml

```diff
@@ -16,7 +16,7 @@ Documentation:
   Enabled: false
 
 LineLength:
-  Max: 99
+  Max: 110
```

Comment:
As I told you before, you shouldn't change this. If you need to disable the line length check of certain line, you can do it
```ruby
very_long_line # rubocop:disable Metrics/LineLength
```